kubernetes/helm/functional-testing/values/test.yaml

```diff
@@ -8,3 +8,19 @@ network: 'russianhill'
 
 namespace: default
 
+networks:
```

Comment:
I would like this file to be network specific (or environment specific). As cent nodes need to be configured to talk to a specific network at start up time, and we have different bootnodes per environment. 
So `test-rinkeby.yaml` and `test-kovan.yaml`  would be the names or using the `russianhill.yaml` or `bernalheights.yaml` if we agree on those names as environments.

So the format for `russianhill.yaml` would be as follows:
```
nodes: 'https://35.184.66.29:8082,https://35.184.39.100:8082'
network: 'russianhill'
namespace: default // potentially this will change in the future
contractAddresses:
  paymentObligation: "0xdb0581A9328664855328AdDb0E251184640f9e5D"
```
and `bernalheights.yaml`:
```
nodes: 'https://35.184.66.29:8082,https://35.184.39.100:8082' // Change node ips
network: 'bernalheights'
namespace: default  // potentially this will change in the future
contractAddresses:
  paymentObligation: "0x0417eb37941164368401D666984cED7694ABcBb1"
```